net/net.c

```diff
@@ -6318,18 +6325,15 @@ int net_set_max_bytes(netinfo_type *netinfo_ptr, uint64_t x)
 int net_sanctioned_list_ok(netinfo_type *netinfo_ptr)
```

Comment:
This looks wrong. You need to break on the first bad node found. Otherwise, answer is the state of the last node in sanc list.

Reply:
Oh wait, you made it harder to read when you rolled that into the for-loop ;)

Reply:
```suggestion
int net_sanctioned_list_ok(netinfo_type *netinfo_ptr)
```

Reply:
Sorry, will move the ok before the ptr check.